src/gosling-track/gosling-track.ts

```diff
@@ -144,43 +175,34 @@ function GoslingTrack(HGC: import('@higlass/types').HGC, ...args: any[]): any {
             this.fetchedTiles = {};
             this.tileSize = this.tilesetInfo?.tile_size ?? 1024;
 
-            // This is tracking the xScale of an entire view, which is used when no tiling concepts are used
-            this.drawnAtScale = HGC.libraries.d3Scale.scaleLinear();
-            this.scalableGraphics = {};
-
             const { valid, errorMessages } = validateTrack(this.options.spec);
 
             if (!valid) {
                 console.warn('The specification of the following track is invalid', errorMessages, this.options.spec);
             }
 
-            this.extent = { min: Number.MAX_SAFE_INTEGER, max: Number.MIN_SAFE_INTEGER };
-
             // Graphics for highlighting visual elements under the cursor
-            this.pMouseHover = new HGC.libraries.PIXI.Graphics();
-            this.pMouseSelection = new HGC.libraries.PIXI.Graphics();
             this.pMain.addChild(this.pMouseHover);
             this.pMain.addChild(this.pMouseSelection);
 
-            // Brushes on the color legend
-            this.gLegend = HGC.libraries.d3Selection.select(this.context.svgElement).append('g');
-
             // Enable click event
-            this.isRangeBrushActivated = false;
             this.pMask.interactive = true;
-            this.gBrush = HGC.libraries.d3Selection.select(this.context.svgElement).append('g');
             this.mRangeBrush = new LinearBrushModel(this.gBrush, HGC.libraries, this.options.spec.style?.brush);
             this.mRangeBrush.on('brush', this.onRangeBrush.bind(this));
-            this.pMask.mousedown = (e: InteractionEvent) =>
-                this.onMouseDown(
-                    e.data.getLocalPosition(this.pMain).x,
-                    e.data.getLocalPosition(this.pMain).y,
-                    e.data.originalEvent.altKey
-                );
-            this.pMask.mouseup = (e: InteractionEvent) =>
-                this.onMouseUp(e.data.getLocalPosition(this.pMain).x, e.data.getLocalPosition(this.pMain).y);
-            this.pMask.mousemove = (e: InteractionEvent) => this.onMouseMove(e.data.getLocalPosition(this.pMain).x);
-            this.pMask.mouseout = () => this.onMouseOut();
```

Comment:
These were all type errors. Looking through the `pixi` docs, the `.on` listeners seem to be the preferred syntax. This is nice because it also mirrors our `LinearBrushModel.on` listener api!